system-addon/content-src/components/Search/_Search.scss

```diff
@@ -3,14 +3,22 @@
   $search-height: 36px;
   $search-input-left-label-width: 35px;
   $search-button-width: 36px;
-  $search-glyph-image: url('#{$image-path}glyph-search-16.svg');
-  $search-glyph-size: 20px;
+  $search-glyph-image: url('chrome://browser/skin/find.svg');
```

Comment:
Let's use `chrome://global/skin/icons/autocomplete-search.svg` to also fix #3333

Reply:
Might need to double check the spacing/sizing.